crates/gitbutler-watcher/src/handler/mod.rs

```diff
@@ -181,4 +186,8 @@ impl Handler {
         }
         Ok(())
     }
+    async fn gitbutler_oplog_change(&self, _project_id: ProjectId) -> Result<()> {
+        // TODO: Queue up pushing of data here, if configured
```

Comment:
@schacon we can initiate synchronization of the oplog from here

doing this in the watcher allows us it to be nice and non-blocking for the various commands